package.json

```diff
@@ -49,12 +49,12 @@
         "react-dom": "^16.12"
     },
     "dependencies": {
-        "@blueprintjs/select": "^3.12.2",
+        "@blueprintjs/select": "^3.12.3",
         "react-uid": "^2.2.0"
     },
     "peerDependencies": {
-        "@blueprintjs/core": "^3.26.0",
-        "@material-ui/core": "^4.9.12",
+        "@blueprintjs/core": "^3.26.1",
+        "@material-ui/core": "^4.9.13",
```

Comment:
please use the last minor version for peerDependencies, i still think maybe the major would be favorable (but that depends on the library)